.env.example

```diff
@@ -1,6 +1,14 @@
 HOST=0.0.0.0
 PORT=1337
-APP_KEYS="toBeModified1,toBeModified2"
-API_TOKEN_SALT=tobemodified
-ADMIN_JWT_SECRET=tobemodified
-JWT_SECRET=tobemodified
+APP_KEYS=
+API_TOKEN_SALT=
+ADMIN_JWT_SECRET=
+TRANSFER_TOKEN_SALT=
+DATABASE_HOST=localhost
+DATABASE_PORT=5432
+DATABASE_NAME=cow-cms-db
+DATABASE_USERNAME=cow-cms-user
+DATABASE_PASSWORD=cow-cms--db-password
+NODE_ENV=development
+DATABASE_CLIENT=postgres
+DATABASE_SSL=false
```

Comment:
nit: can we group them and add some comments?  maybe would make it more self explanatory

Reply:
I mentioned this, because is not obvious all of the parameters, so its always nice to add some documentation here

Reply:
Added.